README.md

```diff
@@ -54,9 +54,9 @@ Files:
 - `NN_weights_YOG_convection.nc` - NetCDF file with weights for the neural net
 - Makefile - Makefile to compile these files
 
-Note that you will need to generate the SAM sounding (an [atmospheric sounding](https://en.wikipedia.org/wiki/Atmospheric_sounding) that defines the SAM grid that the parameterisation is operating on -- 
-we need this grid for the parameterisation, but also to interpolate any inputs from a different model onto this grid so that they can be used in the neural net) as a NetCDF file from the data files in `resources/` if you are using the CAM interface.\
-This can be done from within the `resources/` directory as follows:
+Note that you will need to generate the SAM sounding (an [atmospheric sounding](https://en.wikipedia.org/wiki/Atmospheric_sounding) that defines the SAM grid that the parameterisation is operating on --
+we need this grid for the parameterisation, but also to interpolate any inputs from a different model onto this grid so that they can be used in the neural net) as a NetCDF file from the data files in `YOG_convection/rresources/` if you are using the CAM interface.\
```

Comment:
typo 'rresources' -> 'resources'

Reply:
corrected in https://github.com/m2lines/convection-parameterization-in-CAM/pull/80/commits/fa143448df385b90f60d06939b7ea73acd90e443